### ISSUE TITLE
Openweather KS550 geht nicht über xmlrpc Schnittstelle

### ISSUE THREAD

**Issue by steckenpferd** [thkl/Homematic-Virtual-Interface#122]
Leider verhält sich das HVL Addon nicht wie z.B. CuxD - sprich ich erhalte keine Daten über die xmlrpc Schnittstelle? Irgendwelche Hinweise wie ich die Daten darüber auslesen könnte?

Danke

**Comment by thkl** [comment]
HVL verhält sich wie ein Interface und versteht die meisten xmlrpc Commands. Sonst würde ja auch Rega nicht damit reden. 
Der Port, auf dem das läuft ist 7000 ...

**Comment by steckenpferd** [comment]
![B7803373-F6C6-442C-937D-6D3E05A380C1](https://user-images.githubusercontent.com/39061622/81584663-51dba680-93b3-11ea-92af-a514930652e0.jpeg)

@thkl 7000 klappt nicht

Intern kommuniziert der rega ja auf der CCU.. ja lokal auf der CCU gehen die XML-RPC aufrufe auf port 8301 bei mir.

Sber nicht, wenn ich es z.b. einfach über Postman versuche aufzurufen, da ist der port blockiert, als ob der lighthttp den nicht durchlässt.

**Comment by thkl** [comment]
stimmt, auf der RM CCU selber ist das 8301 .. kann sein, das die RM diesen Port blockert. Ich schreibe nichts in die lighthttp Proxy Config, daher denke ich mal die Firewall blockiert den ggf.

Aber ansonsten : xmlrpc

![image](https://user-images.githubusercontent.com/884355/81585620-b5b29f00-93b4-11ea-97c1-7480e84a5077.png)
